app/views/exclusion/ExclusionOverview.scala.html

```diff
@@ -22,79 +22,120 @@
         govukLayoutWrapper: GovukLayoutWrapper,
         config: AppConfig,
         localFormPartialRetriever: LocalFormPartialRetriever,
-        uriInformation: URIInformation
+        uriInformation: URIInformation,
+        govukRadios: GovukRadios,
+        formWithCSRF: FormWithCSRF
 )
 
 @(
         taxYearRange: TaxYearRange,
         isCurrentTaxYear: String,
         iabdType: String,
         current: List[EiLPerson],
-        empRef: EmpRef
+        empRef: EmpRef,
+        form: Form[MandatoryRadioButton]
 )(implicit request:Request[_], messages: Messages)
 
-@govukLayoutWrapper(pageTitle = messages("ExclusionOverview.title") + " - " + messages("Service.title"), empRef = Some(empRef.toString)) {
+@govukLayoutWrapper(pageTitle = if(current.isEmpty){messages("ExclusionOverview.notExcludedEmployee.title")}else{messages("ExclusionOverview.excludedEmployee.title")} + " - " + messages("Service.title"), empRef = Some(empRef.toString)) {
 
     @defining( if(isCurrentTaxYear=="cy"){ taxYearRange.cyminus1 } else { taxYearRange.cy }){ yearvalue =>
 
-        <h1 id="title" class="govuk-heading-xl">@messages("ExclusionOverview.title")</h1>
+        @if(form.hasErrors){
+            <div class="govuk-error-summary" role="alert" aria-labelledby="error-summary-heading" tabindex="-1" data-module="govuk-error-summary">
+                <h2 class="govuk-error-summary__title" id="error-summary-heading">@messages("Service.errorSummary.heading")</h2>
+                <div class="govuk-error-summary__body">
+                    <ul class="govuk-list govuk-error-summary__list">
+                        <li>
+                            <a href="#error-list-no-selection">@messages("ExclusionOverview.error.required")</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
```

Comment:
is there no reusable component for this in the `frontend-hrmc` package (or similar)?

Reply:
it is there.. but we have to do lot of changes because the existing PBIK implementation is like that... so that I used the html